GliaWidgets/Public/Glia/Glia.swift

```diff
@@ -138,6 +138,13 @@ public class Glia {
         startObservingInteractorEvents()
     }
 
+    /// Minimizes engagement view if ongoing engagement exists.
+    /// Use this function for hiding the engagement view programmatically during ongoing engagement.
+    /// If do so, the operator bubble appears.
```

Comment:
`If you do so` maybe? I don't know exactly what this line is trying to say.

Reply:
1d9a58150126c03cd6034fff3bc17a68c370edcb